src/auth-service/config/firebase-admin.js

```diff
@@ -113,3 +115,5 @@ initializeApp({
 //     logger.error(`internal server error -- ${error.message}`);
 //   }
 // });
+
```

Comment:
Thanks @BenjaminSsempala , I hope we also test out all the rest of the corresponding firebase functionalities just to ensure that things are still functional.

Reply:
Thanks @Baalmart 
The firebase function is also working okay.